### PR TITLE
Create Code Snippets for MSBuild

### PR DESCRIPTION
**Pull request by doggy8088**
Here are my initial code snippets for MSBuild common tasks. I'll add more when I have more spare time.

Complete this issue: https://github.com/tintoy/msbuild-project-tools-vscode/issues/28

### CODE REVIEW

**Comment by doggy8088** [comment]
Here are my initial code snippets for MSBuild common tasks.   I'll add more when I have more spare time.

**Comment by tintoy** [comment]
Thanks! Looking over it now :)

File: snippets/msbuild-project.json [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+		"body": [
+			"<ItemGroup>",
+			"    <DotNetCliToolReference Include=\"Microsoft.DotNet.Watcher.Tools\" Version=\"${1:2.0.0}\" />",
+			"</ItemGroup>",
+			""
+		],
+		"description": "Load DotNetCliToolReference for Microsoft.DotNet.Watcher.Tools"
+	},
+	"Set to IsTransformWebConfigDisabled": {
+		"prefix": "msbuild-dotnet-watcher-tools",
```
Comment (tintoy) [inline snippets/msbuild-project.json]: I think this prefix might be a duplicate of the previous snippet's prefix?

**Comment by tintoy** [comment]
These look great!

I was thinking maybe it would be good if these snippets only show up in MSBuild project files (rather than _all_ XML files) - what do you think? If you agree, and while you're in there, would you mind removing these 3 lines from `package.json`?

https://github.com/tintoy/msbuild-project-tools-vscode/blob/master/package.json#L226-L229

**Comment by tintoy** [comment]
That was quick! :)

**Comment by doggy8088** [comment]
@tintoy I think you can take over it. 😄

**Comment by doggy8088** [comment]
If these snippets only show up in MSBuild project files, that will be awesome.  But I don't know it's possible or not.  The VSCode looks like not support MSBuild file only snippets.

**Comment by tintoy** [comment]
The way snippets work (at least in an extension's `package.json` file), they are [declared per-language](https://github.com/tintoy/msbuild-project-tools-vscode/blob/master/package.json#L222-L231). We have a special language called "MSBuild" (LanguageId = "msbuild"), which is a specialisation of the XML language (so we can do that quite easily). To try it out, see [this section](https://github.com/tintoy/msbuild-project-tools-vscode/#file-extensions) of the README (that's also how you enable stuff like intellisense and syntax highlighting for MSBuild expressions).

**Comment by tintoy** [comment]
Thanks, @doggy8088! I'm going to merge this into a feature branch now and will publish a new release over the weekend once I've had time to fully integrate your changes.

I'll let you know once there's a new VSIX package you can try out (to let me know if you're happy with the result).

**Pull request merged**